curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b566131faa8050a13275b.md

```diff
@@ -0,0 +1,110 @@
+---
+id: 646b566131faa8050a13275b
+title: Step 19
+challengeType: 0
+dashedName: step-19
+---
+
+# --description--
+
+You will create a third `option` HTML element with the `value` attribute set to `mm-dd-yyyy-h-mm` and set the Text inside to `Month, Day, Year, Hours, Minutes`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed